### PR TITLE
add 'searchWorkflow' command

### PR DESCRIPTION
**Pull request by theBaffo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,6 +57,39 @@ const createResponse = (responseMap, text) => {
...
+  response.quickReply.items = items;
+
+  if (items.length === 0) {
+    response.text = "We couldn't find any workflow that matched the search criteria";
+  }
+
+  return [response];
+}
+
+const searchWorkflow = async (client, replyToken, userId, keyword) => {
```
Comment (warunyoud) [inline index.js]: Let's have this function just search the workflow as the name suggests. So it should return the workflows.

File: index.js [hunk 2]
```diff
@@ -81,6 +116,11 @@ async function middleware (params, path) {
           if (/\/send\s\w+\s\w+\s\w+$/.test(message.text)) {
             const [_, text, gid, tid] = message.text.split(' ');
             clients[path].pushV2(gid, tid, createResponse(responseMap, text));
+          } else if (/\/searchWorkflow\s\w+$/.test(message.text)) {
+            const [_, keyword] = message.text.split(' ');
+            const userId = event.source.userId;
+
+            searchWorkflow(clients[path], replyToken, userId, keyword)
```
Comment (warunyoud) [inline index.js]: Let's await the search function here to get the workflows and put the reply code after the function completes.

Comment (theBaffo) [reply index.js]: Done! You can have a look at the new commit.

I had to add a try/catch to be sure that we catch the error from axios.

File: index.js (hunk 1 above)
Comment (theBaffo) [reply index.js]: Done! You can have a look at the new commit.

**Pull request merged**